deploy-sandpit.sh

```diff
@@ -119,12 +120,16 @@ if [[ $BUILD == "1" ]]; then
   echo "done!"
 fi
 
-echo -n "Getting Terraform variables from SSM ... "
-VARS="$(aws ssm get-parameters-by-path --region eu-west-2 --with-decryption --path "/sandpit-deploy/terraform-variables" | jq -r '.Parameters[] | @base64')"
-for VAR in $VARS; do
-  VAR_NAME="TF_VAR_$(echo "${VAR}" | base64 -d | jq -r '.Name / "/" | .[3]')"
-  echo export "${VAR_NAME}"="$(echo "${VAR}" | base64 -d | jq -r '.Value')"
-done
+# echo -n "Getting Terraform variables from SSM ... "
```

Comment:
Again, could we remove rather than comment, or is there a good reason to keep this in?